sql/creators.sh

```diff
@@ -0,0 +1,12 @@
+#/bin/sh
+[ -z $1 ] && OWNER="like1yney2cqn5qdrlc50yr5l53898ufdhxafqz9gxp" || OWNER=$1
+
+echo "supportee of $OWNER"
+psql nftdev <<SQL
```

Comment:
Do we have a public testnet version for all these nft-dev script?

Reply:
Ok I will update these.